ui/src/app/pages/analysis/new-jupyter-notebook-modal.tsx

```diff
@@ -46,7 +46,9 @@ export const NewJupyterNotebookModal = (props: Props) => {
     if (!!existingNameList) {
       setExistingNotebookNameList(existingNameList);
     } else {
-      getExistingNotebookNames(workspace).then(setExistingNotebookNameList);
+      getExistingJupyterNotebookNames(workspace).then(
```

Comment:
OK to change because it's looking for name collisions, and it's more correct to only check against Jupyter here